src/domain/event/utils.ts

```diff
@@ -1522,3 +1522,10 @@ export const omitSensitiveDataFromEventPayload = (
   payload: CreateEventMutationInput | UpdateEventMutationInput
 ): Partial<CreateEventMutationInput | UpdateEventMutationInput> =>
   omit(payload, ['userEmail', 'userName', 'userPhoneNumber']);
+
+export const showTooltipInstructions = (user?: UserFieldsFragment): boolean =>
+  !!user?.adminOrganizations.length;
```

Comment:
Should superusers be included in this check?

Reply:
I was thinking this as well. Superuser is now included in the check